api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -322,9 +342,12 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
     workspaceAuthService.enforceWorkspaceAccessLevel(
         workspaceNamespace, firecloudWorkspaceName, WorkspaceAccessLevel.WRITER);
 
+    DbUser user = userProvider.get();
+    enforceComputeSecuritySuspension(user);
```

Comment:
Can we move this check ahead of `enforceWorkspaceAccessLevel`?  Then we can save a firecloud call if user is suspended.